File: 백준/Bronze/10809. 알파벳 찾기/알파벳 찾기.java
import java.util.Iterator;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		
		String write;
		
		write = input.next();
		
		int[] arr = new int[26];
		
		for (int i = 0; i < arr.length; i++) {
			arr[i] = -1;
		}
		
		for (int i = 0; i < write.length(); i++) {
			int index = write.charAt(i);
			
			if (arr[index -97] == -1) {
				arr[index-97] = i;
			}
		}
		
		for (int i = 0; i < arr.length; i++) {
			System.out.println(arr[i]);
		}
	}
}

File: 백준/Bronze/10809. 알파벳 찾기/알파벳 찾기.cc
#include <stdio.h>
#include <string.h>

int main() {

	char s[101];
	int arr[26];
	scanf("%s", s);

	for (int i = 0; i < 26; i++)
	{
		arr[i] = -1;
	}
	int length = strlen(s);
	for (int i = 0; i < length; i++)
	{
		int index = s[i] - 'a';
		if (arr[index] == -1) {
			
			arr[index] = i;

		}
	}

	for (int i = 0; i < 26; i++)
	{
		printf("%d ", arr[i]);
	}
	return 0;
}
